google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/MetricsTracer.java

```diff
@@ -167,6 +170,14 @@ private void recordAttemptCompletion(@Nullable Throwable throwable) {
                 RpcMeasureConstants.BIGTABLE_ATTEMPT_LATENCY,
                 attemptTimer.elapsed(TimeUnit.MILLISECONDS));
 
+    if (reportBatchingLatency) {
+      measures.put(RpcMeasureConstants.BIGTABLE_BATCH_THROTTLED_TIME, batchThrottledLatency);
+
+      // Reset batch throttling latency for next attempt. This can't be done in attemptStarted
+      // because batching flow control will add batching latency before the attempt has started.
+      batchThrottledLatency = 0;
```

Comment:
should `reportBatchingLatency` also be reset here?

Reply:
nope, it wont change in a single operation